docker/main/rootfs/etc/s6-overlay/s6-rc.d/go2rtc/run

```diff
@@ -44,10 +44,14 @@ function get_ip_and_port_from_supervisor() {
 }
 
 function set_libva_version() {
-    local ffmpeg_path=$(python3 /usr/local/ffmpeg/get_ffmpeg_path.py)
-    export LIBAVFORMAT_VERSION_MAJOR=$($ffmpeg_path -version | grep -Po "libavformat\W+\K\d+")
+    local ffmpeg_path
+    ffmpeg_path=$(python3 /usr/local/ffmpeg/get_ffmpeg_path.py)
+    LIBAVFORMAT_VERSION_MAJOR=$("$ffmpeg_path" -version | grep -Po "libavformat\W+\K\d+")
+    export LIBAVFORMAT_VERSION_MAJOR
 }
 
+set_libva_version
```

Comment:
This one is a potentially important bug-fix.

`create_config.py` was being called **before** `LIBAVFORMAT_VERSION_MAJOR` was set, and yet it attempts to read it.